docs/references/networks-json.mdx

```diff
@@ -0,0 +1,127 @@
+# Defining custom networks
+
+## Overview
+
+When developing multiple projects simultaneously, it can be beneficial to deploy each project to a different local network for simplicity or to support different development workflows. Custom networks can be configured with `dfx` at both the system-wide and project-specific levels. 
+
+## Defining system-wide networks in `networks.json`
+
+Canisters in any project can be deployed to system-wide networks. System-wide custom networks can be configured in the `networks.json` configuration file for `dfx`.
+
+To locate this file on your local system, use the `dfx` command:
+
+```
+dfx info networks-json-path
+```
+
+This will return the file path where your system's `networks.json` file is located, such as:
+
+```
+/Users/developer/.config/dfx/networks.json
+```
+
+### Default configuration
+
+By default, this file contains the following configuration:
+
+```json
+{
+    "local": {
+        "replica": {
+            "subnet_type": "application"
+        }
+    }
+}
+```
+
+Alternatively, if a `network.json` file is not defined locally, `dfx` will by default use the above configuration. 
+
+### Defining a custom network
+
+To define a custom network, you can define a binding address in your `networks.json` to be `localhost` or any other domain name, such as:
+
+```json
+{
+  "network_1": {
+    "bind": "localhost:4943",
+    "replica": {
+      "subnet_type": "application"
+    }
+  }
+}
+```
+
+To define multiple networks, use multiple definitions on different domain names:
+
+```json
+{
+  "network_1": {
+    "bind": "localhost:4943",
+    "replica": {
+      "subnet_type": "application"
+    }
+  },
+    "network_2": {
+    "bind": "127.0.0.1:4943",
+    "replica": {
+      "subnet_type": "application"
+    }
```

Comment:
Identation is a bit off for `network_2`.

Reply:
Fixed indentation